src/dotnetTemplate/Microsoft.PowerShell.PowerShellStandard.Template/Microsoft.PowerShell.PowerShellStandard.Template.nuspec

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.PowerShell.PowerShellStandard.Template</id>
```

Comment:
I think Module should be in the id as this isn't for PowerShell Standard itself.  Maybe Microsoft.PowerShell.Standard.Module.Template